<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceQueryWidget</class>
 <widget class="QWidget" name="ResourceQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>248</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Please enter a SPARQL query:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_buttonBack">
       <property name="toolTip">
        <string>Browse back in the query history</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_buttonForward">
       <property name="toolTip">
        <string>Browse forward in the query history</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_queryButton">
       <property name="text">
        <string>Run Query</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_stopQueryButton">
       <property name="text">
        <string>Stop Query</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QueryEditor" name="m_queryEdit" native="true"/>
     <widget class="QTableView" name="m_queryView"/>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_statusLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QueryEditor</class>
   <extends>QWidget</extends>
   <header>queryeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
